Google.Cloud.AspNetCore/Google.Cloud.AspNetCore.DataProtection.Kms/KmsXmlConstants.cs

```diff
@@ -24,21 +24,24 @@ internal static class KmsXmlConstants
         /// <summary>
         /// The name of the outer element.
         /// </summary>
-        internal static XName EncryptedElement { get; } = "encryptedKey";
+        internal static XName EncryptedElement { get; } = "encryptedData";
 
         /// <summary>
-        /// The name of the attribute within <see cref="EncryptedElement"/> specifying the key name.
+        /// The name of the attribute within <see cref="EncryptedElement"/> specifying the KMS key name
+        /// used to encrypt the local key data.
         /// </summary>
-        internal static XName KeyNameAttribute { get; } = "keyName";
+        internal static XName KmsKeyNameAttribute { get; } = "kmsKeyName";
 
         /// <summary>
-        /// The name of the element within <see cref="EncryptedElement"/> specifying the encrypted value.
+        /// The name of the attribute within <see cref="EncryptedElement"/> specifying the encrypted
+        /// form of the local key data.
         /// </summary>
-        internal static XName ValueAttribute{ get; } = "encryptedValue";
+        internal static XName LocalKeyDataAttribute { get; } = "localKeyData";
 
         /// <summary>
-        /// The maximum size of data that we expect to be able to encrypt.
+        /// The name of the element within <see cref="EncryptedElement"/> specifying the value that has
+        /// been encrypted with the local key.
         /// </summary>
-        internal const int MaximumEncryptSize = 65536;
+        internal static XName PayloadElement{ get; } = "encryptedValue";
```

Comment:
One more datum I'd like to see in the XML element: a version number.  Just in case we decide to change the storage format in a future release, it'll be easier to tell the old format from the new format with an explicit version number in the xml element.

Reply:
Hmm... not sure. The whole point of using a proto is that it can evolve in a backwardly-compatible way. So long as we keep generating things by default that the "old" code can read, it doesn't matter if the "new" code can read/generate more. (You'd need to make sure you deploy newest code before you start *using* the functionality of generating data that can't be read by the old code.